src/config.rs

```diff
@@ -94,6 +94,10 @@ create_config! {
     spaces_around_ranges: bool, false, false, "Put spaces around the  .. and ..= range operators";
     binop_separator: SeparatorPlace, SeparatorPlace::Front, true,
         "Where to put a binary operator when a binary expression goes multiline";
+    fn_generics_space: FnGenericsSpace, FnGenericsSpace::Never, false,
+        "Leave spaces around function generics.";
+    fn_no_generics_space: bool, false, false,
+        "Leave a space after the function name when it's not generic.";
```

Comment:
I always worry I'll hit some line length/wrapping issues with inline suggestions, but maybe we could use something like this?

```suggestion
        "Whether to put a space between the function name and opening paren in signatures without generics";
```

Reply:
Fixed